packages/react-badge/src/BadgeContent.stories.tsx

```diff
@@ -84,3 +84,8 @@ export const BadgeContentExample = () => (
     </div>
   </div>
 );
+
+export default {
+  title: 'Components/BadgeContent',
+  component: Badge,
```

Comment:
nice I was just about to write you that this needs to be tweaked as well. good job. thx!